src/files.ts

```diff
@@ -0,0 +1,127 @@
+import { CID } from "multiformats/basics.js";
+import importer, {
+  Options as AddOptions,
+  File,
+  Result,
+} from "ipfs-unixfs-importer";
+import { Buffer } from "buffer";
+import exporter, { Options as GetOptions } from "ipfs-unixfs-exporter";
+import { Peer } from "./core";
+import Ipld from "ipld";
+
+// Export the interfaces for use by callers
+export { Options as AddOptions, File, Result } from "ipfs-unixfs-importer";
+export { Options as GetOptions } from "ipfs-unixfs-exporter";
+
+// Module augmentation to add methods to Peer from core
+declare module "./core" {
+  interface Peer {
+    addFile(
+      source: Iterable<File>,
+      options?: AddOptions
+    ): Promise<Result | undefined>;
+    getFile(cid: CID | Buffer | string, options?: GetOptions): Promise<Buffer>;
+  }
+}
+
+// @todo: Just support this API directly? Required to support the 'Files' style APIs.
+const wrap = (ipld: Peer) => {
```

Comment:
This wrapper was required to support some up/downstream updates to js-ipfs stuff. Just using this wrapper to match the expected API rather than anything serious, as our APIs might change soon anyway to avoid worrying about API compat.